modules/date.js

```diff
@@ -0,0 +1,21 @@
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+const displayDate = () => {
+  const dateElement = document.querySelector('.date');
+  const dt = DateTime.now();
+
+  const options = {
+    month: 'long',
+    day: 'numeric',
+    year: 'numeric',
+    hour: '2-digit',
+    minute: '2-digit',
+    second: '2-digit',
+  };
+
+  const all = dt.toLocaleString(options);
+
+  dateElement.textContent = `${all}`;
+};
+
+export default displayDate;
```

Comment:
- [ ] The date is not displayed below the navbar. Kindly work on that.
<img width="946" alt="image" src="https://user-images.githubusercontent.com/77190203/159340091-6b1e35e7-be21-4db3-8110-e82868858e97.png">